pkg/telemetery/settings.go

```diff
@@ -0,0 +1,15 @@
+package telemetery
+
+import (
+	"github.com/jaegertracing/jaeger/pkg/healthcheck"
+	"github.com/jaegertracing/jaeger/pkg/jtracer"
+	"github.com/jaegertracing/jaeger/pkg/metrics"
+	"go.uber.org/zap"
+)
+
+type Setting struct {
+	Logger  *zap.Logger
+	Tracer  *jtracer.JTracer
```

Comment:
```suggestion
	TracerProvider  otel.TracerProvider
```

Reply:
I will be implementing that change in some time, after I am done changing the HC to use OTEL's Report Status functionality instead.